src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -36,28 +40,52 @@ const useStyles = makeStyles((theme: Theme) => ({
   expandOpen: {
     transform: 'rotate(180deg)',
   },
+  subMenu: {
+    backgroundColor: theme.palette.cruGrayDark.main,
+  },
+  menuItem: {
+    color: 'white',
+    '&:hover': {
+      backgroundColor: theme.palette.cruGrayMedium.main,
+    },
+  },
+  menuItemSelected: {
+    backgroundColor: theme.palette.cruGrayMedium.main,
+  },
+  menuIcon: {
+    marginRight: theme.spacing(1),
+  },
 }));
 
 const NavMenu = (): ReactElement => {
   const { t } = useTranslation();
   const classes = useStyles();
   const accountListId = useAccountListId();
+  const currentToolId = useCurrentToolId();
 
   const [reportsMenuOpen, setReportsMenuOpen] = useState(false);
+  const [toolsMenuOpen, setToolsMenuOpen] = useState(false);
   const anchorRef = React.useRef<HTMLLIElement>(null);
+  const anchorRefTools = React.useRef<HTMLLIElement>(null);
 
   const handleReportsMenuToggle = () => {
     setReportsMenuOpen((prevOpen) => !prevOpen);
+    handleToolsMenuClose();
   };
 
-  const handleReportsMenuClose = (event: React.MouseEvent<EventTarget>) => {
-    if (anchorRef.current?.contains(event.target as HTMLElement)) {
```

Comment:
I'm not sure how important this is but in order to get the codecov to pass I removed this since idk how to test this line. Everything still seems to work fine

Reply:
Seems like maybe this was to prevent the menu from closing when something inside it was clicked? Maybe there was a submenu at one point. Idk. Seems to behave fine. Idk that I see a reason for this either.